src/common/Defs.h

```diff
@@ -53,8 +53,6 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define MASTER4_SERVER_NAME ""
 #define MASTER5_SERVER_NAME ""
 
-#define WWW_BASEURL         "dl.unvanquished.net/pkg"
```

Comment:
What's wrong with this default? Also if you are going to change the default, you should at least provide a note and rationale in the commit message.

The commit message suggests this change has no functional impact.

Reply:
oh, nothing is wrong. I only forgot to put it back in the value field :S